### ISSUE TITLE
Create preview build

### ISSUE THREAD

**Issue by stevepiercy** [plone/plone-sphinx-theme#4]

**Comment by stevepiercy** [comment]
Uh-oh. https://answers.netlify.com/t/the-current-python-version-3-8-is-very-old/96966/4

> To answer your question, yes there are some technical limitations due to which we’re sticking on Python 3.8. Allowing newer versions would require a bigger engineering lift and that’s not something prioritised in the short-term. Seeing your question, I did ask out devs about the status, and it’s still “not planned in any short-term roadmap”.

Meanwhile:

https://fosstodon.org/@orsinium/110807654189402872

> I just migrated pythonetc.orsinium.dev from [#netlify](https://fosstodon.org/tags/netlify) to [#cloudflare](https://fosstodon.org/tags/cloudflare) pages.
> 
> Cloudflare pros: Python 3.11 (Netlify is still on 3.8 and no plans to ever upgrade), amazing analytics, unlimited free bandwidth.
> 
> Netlify pros: more configuration options, has a config file, friendlier UI.
> 
> Both provide PR previews and are easy to use and fast to set up.

**Comment by stevepiercy** [comment]
We need to migrate from Netlify or stop using it. Sphinx Book Theme requires Python 3.9+, and Netlify supports only Python 3.8. 💩 

https://app.netlify.com/sites/plone-sphinx-theme/deploys/663b5147d6be0c00083499ad#L218

**Comment by stevepiercy** [comment]
Looks like Cloudflare Pages is the way to go. Example: https://github.com/life4/pythonetc/pull/53#issuecomment-1679016218

**Comment by stevepiercy** [comment]
Maybe not. I had no luck in #7. I looked at Vercel, and got overwhelmed.

I also looked at https://github.com/marketplace/actions/deploy-pr-preview, but this works only with GitHub Pages classic and a specific branch, not the latest version through GitHub Actions. See https://github.com/rossjrw/pr-preview-action/issues/21#issuecomment-1824112900.

GitHub has this feature that they have in alpha and for their internal use only for the last two years, but it is unknown if or when it will become public. See https://github.com/orgs/community/discussions/7730#discussioncomment-7603270.

**Comment by stevepiercy** [comment]
It looks like Read The Docs is possible, either through GHA or a webhook. I asked in the Python Docs Discord.

https://github.com/readthedocs/actions/tree/v1/preview

Before I begin, does the Plone Foundation have a RTD organization with subprojects?

Pylons Project has one.

![Screenshot 2024-05-09 at 2 56 46 PM](https://github.com/plone/plone-sphinx-theme/assets/102112/a64e9062-8727-48c7-8907-9c903230eb11)

Here's breadcrumbs of Pyramid.

![Screenshot 2024-05-09 at 2 59 04 PM](https://github.com/plone/plone-sphinx-theme/assets/102112/92231464-4c83-467a-bc1a-cf6f49e10bfc)

It looks like plone.restapi is a single organization-less project.

![Screenshot 2024-05-09 at 2 58 53 PM](https://github.com/plone/plone-sphinx-theme/assets/102112/67877547-dffd-42ce-8475-56fe95c69211)

@polyester @tisto @ericof @sneridagh @tkimnguyen @mauritsvanrees @fredvd might any of you know? If there is none, should we set one up?

**Comment by stevepiercy** [comment]
We might not actually want an organization with sub-projects because we cannot have custom domains for each one. The pattern is:

```
custom.plone.org/projects/<subproject>/
```

With each project as a plain old project, we can do this:

```
<project>.plone.org
```

There are some advantages to subprojects, including being able to search across them and to use a single domain name for each, but I don't think those features are worth it.

Anyway, for now, we have https://plone-sphinx-theme.readthedocs.io/.

We could set up a custom domain of https://plone-sphinx-theme.plone.org/. @polyester @fredvd what are your thoughts about doing this?

**Comment by stevepiercy** [comment]
Closed by #11
